build.sbt

```diff
@@ -431,9 +445,21 @@ lazy val unidocSettings = Seq(
   ),
   JavaUnidoc / unidoc / unidocAllSources := (JavaUnidoc / unidoc / unidocAllSources).value.map {
     sources =>
-      sources.filterNot { source =>
-        source.toString.contains("$") || source.toString.contains("packageprivate")
-      }
+      sources
+        .filterNot { source =>
+          source.toString.contains("$") || source.toString.contains("packageprivate")
+        }
+        .filter { source =>
+          //
+          // This filter limits javadoc to only files with
+          // api, sapi, or japi directories in their paths.
+          //
+          val str = source.toString
```

Comment:
The generated javadoc doesn't contain the metadata an other classes mentioned above. I think because they are written in scala so would need the genjavadoc plugin. Or do we want to exclude those and consider them experimental? Maybe we should only allow "/layers/api/" or something?